### PR TITLE
Handling of Discogs/nonLD search results and retrieving term.

### PR DESCRIPTION
**Pull request by justinlittman**

### CODE REVIEW

**Comment by justinlittman** [comment]
This moves discogs support forward; however, QA is currently returning invalid N3 for discogs resources.

**Comment by justinlittman** [comment]
Unfortunately, getting all of the necessary RTs is problematic. Ping me and I can demo via slack.

File: src/components/search/QASearchResults.jsx [hunk 1]
```diff
@@ -22,15 +22,17 @@ const QASearchResults = (props) => {
...
   const [resourceTemplateId, setResourceTemplateId] = useState(null)
   const [resourceN3, setResourceN3] = useState(null)
   const [resourceState, unusedDataset, useResourceError] = useResource(resourceN3, resourceURI, resourceTemplateId, rootResource, props.history)
 
-  useEffect(() => {
-    if (useResourceError && !error) {
-      setError(useResourceError)
-    }
-  }, [useResourceError, error])
+  // useEffect(() => {
```
Comment (jermnelson) [inline src/components/search/QASearchResults.jsx]: Cruft?

Comment (justinlittman) [reply src/components/search/QASearchResults.jsx]: Decrufted @jermnelson

Comment (jermnelson) [reply src/components/search/QASearchResults.jsx]: 👍

**Pull request merged**